functions/__fzf_search_git_log.fish

```diff
@@ -10,12 +10,12 @@ function __fzf_search_git_log --description "Search the git log of the current g
             # see documentation for git format placeholders at https://git-scm.com/docs/git-log#Documentation/git-log.txt-emnem
             # %h gives you the abbreviated commit hash, which is useful for saving screen space, but we will have to expand it later below
             git log --color=always --format=format:'%C(bold blue)%h%C(reset) - %C(cyan)%as%C(reset) %C(yellow)%d%C(reset) %C(normal)%s%C(reset)  %C(dim normal)[%an]%C(reset)' | \
-            fzf --ansi --tiebreak=index --preview='git show --color=always (string split --max 1 " " {})[1]'
+            fzf --ansi --tiebreak=index --preview='git show --color=always (string split --max 1 " " {})[1]' --query=(commandline --current-token)
         )
         if test $status -eq 0
             set abbreviated_commit_hash (string split --max 1 " " $selected_log_line)[1]
             set commit_hash (git rev-parse $abbreviated_commit_hash)
-            commandline --insert $commit_hash
+            commandline --current-token --replace $commit_hash
```

Comment:
Is there really any value to this? I guess it doesn't matter either way but I can't see anyone memorizing git-shas and starting to type them and then triggering this function. Do you do that?

Reply:
- This is consistent with other functions
- You don't have to always type SHA. Let's say I want to `git checkout dev|` (| is the cursor), and I realized I actually want to go to some other commit that mentioned `dev`, so I can trigger it and don't have to type the same thing again.

Reply:
good point, hadn't considered that use case